recipe/meta.yaml

```diff
@@ -220,6 +224,7 @@ outputs:
         - if not exist %LIBRARY_BIN%/liblapacke.dll exit 1           # [win]
 
   - name: blas-devel
+    # uses lapack {{ version }}, not {{ blas_major }}
```

Comment:
I was double-checking this, because it's not immediately clear from the recipe what's happening here. To be honest, I was surprised that `blas-devel` is using a different versioning scheme than `blas`, but whether intentional or not, it probably can't be fixed or changed until `blas_major` overtakes the lapack version. In the meantime, I thought it's better to leave a comment.